test/express/middleware-test.js

```diff
@@ -20,6 +20,19 @@ describe('input-validation middleware tests - Express', function () {
             middleware.init('test/pet-store-swagger.yaml');
         });
     });
+    describe('initAsync function tests', async function () {
+        it('should throw an error in case the file doesn\'t exists', function () {
+            const middleware = require('../../src/middleware');
+            expect(async () => {
+                await middleware.initAsync('test/pet-store-swagger1.yaml');
+            }).to.throw;
+        });
+        it('should resolve without formats', async function () {
+            const rewire = require('rewire');
```

Comment:
Why is rewire needed here?

Reply:
this test is basically the same test as the test for the normal init with modifications to work with async function.
I don't understand the why it was part of the original test so I didn't want to remove it.